packages/ember-views/lib/views/view.js

```diff
@@ -994,6 +995,29 @@ var View = CoreView.extend(
   },
 
   /**
+    @private
+
+    Renders the view into `body` element and returns the element.
+
+    This method is useful if you want to render the view into an element that
+    is not in the document's body. Instead, a new `body` element, detached from
+    the DOM is returned. FastBoot uses this to serialize the rendered view into
+    a string for transmission over the network.
+
+    @method renderToElement
+    @return {HTMLBodyElement} element
+  */
+  renderToElement: function() {
+    var element = this.renderer._dom.createElement('body');
+
+    run(this, function() {
+      this.renderer.appendTo(this, element);
+    });
```

Comment:
@mixonic I removed the run loop, as requested. However, the thing to keep in mind is that in nearly all instances you will be calling this method from inside a resolved `visit()` promise, which is already inside a run loop. In most cases you will likely want to synchronously render/serialize the element, as we do in these tests.

It's easy enough to wrap in a helper (https://github.com/emberjs/ember.js/pull/10424/files#diff-083b73026ec5128ae78dc823a24e44dcR77) but I'm afraid this might be a hard-to-debug mistake that many people make. I'm fine with it for now though.